app/models/miq_report.rb

```diff
@@ -224,4 +224,16 @@ def remove_loading_relations_for_virtual_custom_attributes
     vc_attributes = CustomAttributeMixin.select_virtual_custom_attributes(cols).present?
     include.delete(:custom_attributes) if vc_attributes.present? && include && include[:custom_attributes].blank?
   end
+
+  # determine name column from headers for x-axis in chart
+  def chart_header_column
+    if graph[:column].blank?
+      _log.error("Column for x-axis of chart have to be defined in report")
```

Comment:
Message text should be `The column for the chart's x-axis must be defined in the report`.